app/views/dashboards/show.html.erb

```diff
@@ -1,13 +1,66 @@
-<script src="./assets/javascripts/vextab-div.js"></script>
-
 <div id='dashboard-header'>
   <h1>Welcome to the Music Snippets dashboard!</h1>
   <%= link_to "Save a snippet", new_snippet_path %>
   <%= link_to "Save an audio file", new_audio_file_path %>
   <%= link_to "Save a wishlist item", new_wishlist_path %>
-  <h2>🎹 Latest music snippet activity: 🎸</h2>
 </div>
 
+<hr>
+<h1>🎹🎸 Latest activity:</h1>
 <div class="snippet">
   <%= render @activities %>
 </div>
+
+<hr>
+<h1>🎹🎸 Music suggestions:</h1>
+
+<script type="text/javascript">
+
+function getJobStatus(jobId) {
+  $.ajax({
+    url: "/suggestions_job_statuses/" + jobId,
+    type: "GET",
+    dataType: "json",
+    error: function(xhr, errorString, exceptionObj) {
+      console.log(errorString);
+      console.log(xhr);
+      console.log(exceptionObj);
+      console.log("so sad");
+    },
+    statusCode: {
+      200: function (data, status, xhr) {
```

Comment:
I think the 202 header, `Accepted` might make most sense to say "Something is in progress, try again soon"

https://httpstatuses.com/202